main/iso-codes/APKBUILD

```diff
@@ -1,34 +1,32 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=iso-codes
-pkgver=3.75
-pkgrel=1
-pkgdesc="Lists of the country, language, and currency names"
-url="http://pkg-isocodes.alioth.debian.org/"
+pkgver=3.79
+pkgrel=0
+pkgdesc="ISO codes and their translations"
+url="http://pkg-isocodes.alioth.debian.org"
 arch="noarch"
-license="LGPL-2.0-or-later"
+license="LGPL-2.1-or-later"
```

Comment:
I can't find an `or-later' clause in the source here either on a first sight.

Reply:
So, the files in `bin/` are in fact licensed under `LGPL-2.1-or-later` the `*.po` files only state that they are released `under the same license as the iso-codes package.` which makes me wonder if this means `LGPL-2.1` or `LGPL-2.1-or-later` :thinking:

Reply:
Same. The included license indicates the `*-or-later' clause :

    This library is free software; you can redistribute it and/or
    modify it under the terms of the GNU Lesser General Public
    License as published by the Free Software Foundation; either
    version 2.1 of the License, or (at your option) any later version.